spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/TestEntity.java

```diff
@@ -94,4 +96,36 @@ public void setSize(Long size) {
 	enum Shape {
 		CIRCLE, SQUARE;
 	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
Why is this important? We don't typically add the `equals` and `hashCode` for entities?

Reply:
we use it in tests for parameter matching